### PR TITLE
Update lib version

### PR DESCRIPTION
**Pull request by lbiselli**
Related: https://github.com/inspetor/inspetor-android/pull/21

Just updated the library version to `1.4.0-beta`

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,8 +1,120 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
+    <AndroidXmlCodeStyleSettings>
```
Comment (pearsonhenri) [inline .idea/codeStyles/Project.xml:3]: nit: let's gitignore this file :)

**Review by pearsonhenri — APPROVED** [review]
small nit (feel free to address in a follow-up, but otherwise looks good!)

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (lbiselli) [reply .idea/codeStyles/Project.xml:3]: So I found out that is not best practices to gitignore this file though I found that we were not ignoring some files that we should have been

Comment (pearsonhenri) [reply .idea/codeStyles/Project.xml:3]: hm! I guess it's to keep the Android dev settings in common for developers

**Comment by pearsonhenri** [comment]
LGTM!

**Pull request merged**